docker-compose.storage-squid.yml

```diff
@@ -0,0 +1,79 @@
+version: '3'
+
+services:
+  squid_db:
+    container_name: squid_db
+    hostname: squid-db
+    image: postgres:16
+    restart: unless-stopped
+    environment:
+      POSTGRES_DB: ${SQUID_DB_NAME}
+      POSTGRES_PASSWORD: ${SQUID_DB_PASS}
+    networks:
+      - joystream
+    ports:
+      - '127.0.0.1:${SQUID_DB_PORT}:${SQUID_DB_PORT}'
+    command: ['postgres', '-c', 'config_file=/etc/postgresql/postgresql.conf', '-p', '${SQUID_DB_PORT}']
+    shm_size: 1.1g # A bit higher than `shared_buffers` value in postgres.conf
+    volumes:
+      - squid_db_data:/var/lib/postgresql/data
+      - ./postgres.conf:/etc/postgresql/postgresql.conf
+
+  squid_processor:
+    image: joystream/storage-squid:latest
+    container_name: squid_processor
+    hostname: squid-processor
+    restart: unless-stopped
+    env_file:
+      - .env
+    environment:
+      - DB_NAME=${SQUID_DB_NAME}
+      - DB_PASS=${SQUID_DB_PASS}
+      - DB_PORT=${SQUID_DB_PORT}
+      - DB_HOST=${SQUID_DB_HOST}
+      - GQL_PORT=${SQUID_GQL_PORT}
+      - ARCHIVE_GATEWAY_URL=${SQUID_ARCHIVE_GATEWAY_URL}
+      - RPC_ENDPOINT=${JOYSTREAM_NODE_WS}
+
+    ports:
+      - '127.0.0.1:${PROCESSOR_PROMETHEUS_PORT}:${PROCESSOR_PROMETHEUS_PORT}'
```

Comment:
change to use `SQUID_PROCESSOR_PROMETHEUS_PORT` instead. Otherwise services are failing to start with:

```
Error response from daemon: driver failed programming external connectivity on endpoint squid_processor (e1a9bb40428326391a37414aa61f52dfd4817f99dc1662c278013f4780d7c979): Bind for 127.0.0.1:3337 failed: port is already allocated
```

When we start network with `start.sh` or `start-multistorage.sh` that starts orion services first which is listening on `PROCESSOR_PROMETHEUS_PORT` port.

Reply:
Done in [897c7c0](https://github.com/Joystream/joystream/pull/5001/commits/897c7c0e38110a5fcbadd8cc20a3add82642215c)